data_structures/stacks/stack_with_two_queues.py

```diff
@@ -0,0 +1,68 @@
+from collections import deque
+from typing import Generic, TypeVar
+
+T = TypeVar("T")
+""" Implementing stack using two arrays"""
+
+
+class Stack(Generic[T]):  # Stack class to implement stack operations
+    def __init__(self) -> None:
+        self.insert_queue = deque()  # First Queue to be used for inserting
+        self.suffle_queue = deque()  # Second Queue to be used for suffling
+
+    def push(self, item: int) -> None:
+        self.insert_queue.append(item)  # Add items into the Queue
```

Comment:
The code already says what the comment repeats.
```suggestion
        self.insert_queue.append(item)
```